lib/vizkit/vizkit_items.rb

```diff
@@ -803,6 +803,27 @@ def write(&block)
         end
     end
 
+    class TaskModelItem < VizkitItem
+        attr_reader :model
+
+        def initialize(task, options = Hash.new)
```

Comment:
Sure I can do it, but then all other subclasses of VizkitItem don't. So a 'local' style cleanup would be kinda inconsistent. Similar for caching options and raising exception -- just maintaining previous behaviour.